File: root_VS2012_TemplateBase/programs/VB/Samples/Bat_sample/RerunnableBatch_sample/Common/SelectPkListReturnValue.vb
'**********************************************************************************
'* フレームワーク・テストクラス
'**********************************************************************************

'**********************************************************************************
'* クラス名        ：SelectPkListReturnValue
'* クラス日本語名  ：テスト用の戻り値クラス
'*
'* 作成日時        ：－
'* 作成者          ：生技セ
'* 更新履歴        ：
'*
'*  日時        更新者            内容
'*  ----------  ----------------  -------------------------------------------------
'*  20xx/xx/xx  ＸＸ ＸＸ         ＸＸＸＸ
'*
'**********************************************************************************

' System
Imports System.IO
Imports System.Data
Imports System.Text
Imports System.Collections
Imports System.Collections.Generic

' ベースクラス
Imports Touryo.Infrastructure.Business.Common

Namespace Common
	''' <summary>
	''' SelectPkListReturnValueの概要の説明です
	''' </summary>
	Public Class SelectPkListReturnValue
		Inherits MyReturnValue
		''' <summary>PkList</summary>
		Public PkList As ArrayList
	End Class
End Namespace

File: root_easysetup/programs/C#/Samples/Bat_sample/RerunnableBatch_sample2/Common/SelectPkListReturnValue.cs
﻿//**********************************************************************************
//* フレームワーク・テストクラス
//**********************************************************************************

//**********************************************************************************
//* クラス名        ：SelectPkListReturnValue
//* クラス日本語名  ：テスト用の戻り値クラス
//*
//* 作成日時        ：－
//* 作成者          ：生技セ
//* 更新履歴        ：
//*
//*  日時        更新者            内容
//*  ----------  ----------------  -------------------------------------------------
//*  20xx/xx/xx  ＸＸ ＸＸ         ＸＸＸＸ
//*
//**********************************************************************************

// System
using System;
using System.IO;
using System.Data;
using System.Text;
using System.Collections;
using System.Collections.Generic;

// ベースクラス
using Touryo.Infrastructure.Business.Common;

namespace RerunnableBatch_sample2.Common
{
    /// <summary>
    /// SelectPkListReturnValueの概要の説明です
    /// </summary>
    public class SelectPkListReturnValue : MyReturnValue
    {
        /// <summary>PkList</summary>
        public ArrayList PkList;
    }
}
